packages/semcom-components/lib/components/gender.component.ts

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable no-console -- is a web component */
+import { NamedNode, Store } from 'n3';
+import { css, html, property, PropertyValues } from 'lit-element';
+import { ComponentResponseEvent } from '@digita-ai/semcom-sdk';
+import { BaseComponent } from './base.component';
+
+export class GenderComponent extends BaseComponent {
+
+  @property() gender = 'Unknown';
+
+  /**
+   * Handles a response event. Can be used to update the component's properties based on the data in the response.
+   *
+   * @param event The response event to handle.
+   */
+  handleResponse(event: ComponentResponseEvent): void {
+
+    if (!event || !event.detail || !event.detail.data) {
+
+      throw new Error('Argument event || !event.detail || !event.detail.quads should be set.');
+
+    }
+
+    const genderPrefix = 'http://digita.ai/voc/gender#';
+
+    const store = new Store(event.detail.data);
+
+    this.gender = store.getQuads(null,  new NamedNode(`${genderPrefix}type`), null, null)[0]?.object.value.split('#')[1];
+
+  }
+
+  /**
+   * Is executed when a property value is updated.
+   *
+   * @param changed Map of changes properties.
+   */
+  update(changed: PropertyValues): void {
+
+    super.update(changed);
+
+    if (changed.has('entry') && this.entry) this.readData(this.entry);
+
+  }
+
+  static get styles() {
+
+    return [
+      css`
+        
+      `,
+    ];
+
+  }
+
+  render() {
+
+    const imgUrl = new URL(`../assets/gender/${this.gender}.png`, import.meta.url);
```

Comment:
So, what made this work in the end?

Reply:
I think a bug in "esnext", I changed es2017 to esnext, and inspected the js files of the components, and saw that import.meta was being replaced by a variable "e", which had been assigned the value {} to it

It was fixed when I changed it to es2020. However, according to what I could find, esnext should've worked, but it didn't